pkg/cloudprovider/aws/aws.go

```diff
@@ -403,6 +411,8 @@ InstanceReadyLoop:
 				break InstanceReadyLoop
 			}
 		case <-deadline.C:
+			log.Info("Reached instance ready deadline but not all instances are ready")
+			terminateOrphanedInstances(n, instances)
```

Comment:
`terminateOrphanedInstances` seems to be called before every `return err`, could this not be moved out to https://github.com/atlassian/escalator/blob/25b9823f614d772deb12bcb07ac1e0fe4aa9cd1e/pkg/cloudprovider/aws/aws.go#L258?

Reply:
Resolved as realised terminate instances depending on the api call and instances returned. No action needed